app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p><%= t 'devise.mailer.email_changed.hello'%> <%= @email %>!</p>
+
+<% if @resource.try(:unconfirmed_email?) %>
```

Comment:
`@resource.try(:unconfirmed_email?)` => `@resource&.unconfirmed_email?`

Reply:
redone as you asked!